lesson07/assignment/sp1/src/components/todoitem.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <section>
+    <li>{{ todo.text }}</li>
+
+  </section>
+
+</template>
+
+<script>
+
+export default {
+  name: 'todoitem',//name of file, component, html tag
+  props: {
+    todo: String
+    //my prop name : data type
+    // can also list names
+    // try an array
+  }
+}
```

Comment:
Since this component takes only the "todo" prop and it's of type String, the reference to an attribute of "todo" on line 3 (todo.text) will fail. It would only work if "todo" was an object that included a key of "text".